girder/api/describe.py

```diff
@@ -218,6 +245,87 @@ def param(self, name, description, paramType='query', dataType='string',
         self._params.append(param)
         return self
 
+    def modelParam(self, name, description=None, model=None, destName=None, paramType='path',
+                   plugin='_core', level=None, required=True, force=False, exc=True,
+                   requiredFlags=None, **kwargs):
+        """
+        This should be used in lieu of ``param`` if the parameter is a model ID
+        and the model should be loaded and passed into the route handler.
```

Comment:
It's hard for me to read through the docstring of the params and figure out how to use this with the defaults.  Can a few examples be provided in the docstring, using `name` and `map` and `model`?  With one example using `id` and one using something like `folderId` perhaps.